loleaflet/src/control/Control.MobileWizard.js

```diff
@@ -438,7 +449,13 @@ L.Control.MobileWizard = L.Control.extend({
 			this._builder.build(this.content.get(0), [data]);
 
 			this._mainTitle = data.text ? data.text : '';
-			this._setTitle(this._mainTitle);
+			this._mainTitleHTML = data.html ? data.html : '';
+			this._changeLevelTitle = data.changeLevelTitle !== undefined ? data.changeLevelTitle : true;
```

Comment:
here will be needed just one line:
this._customTitleBar = data.customTitleBar;   (because it will be undefined or will have content to show)